src/webui/www/private/views/rss.html

```diff
@@ -0,0 +1,815 @@
+                        //filter children
+                        let reducedDatapaths = selectedDatapaths.filter(path => 
+                            selectedDatapaths.filter(innerPath => path.slice(0, innerPath.length) == innerPath).length == 1
+                        );
+                        removeItem(reducedDatapaths);
+                    },
+                    newSubscription: addRSSFeed,
+                    newFolder: addFolder,
+                    updateAll: refreshAllFeeds
+                },
+                offsets: {
+                    x: -16,
+                    y: -57
+                }
+            });
+
+            rssFeedContextMenu.addTarget($('rssFeedTableDiv'));
+            //deselect feed when clicking on empty part of table
+            $('rssFeedTableDiv').addEventListener("click", function(e) {
+                rssFeedTable.deselectAll();
+                rssFeedTable.deselectRow();
+            });
+            $('rssFeedTableDiv').addEventListener("contextmenu", function(e) {
+                if (e.toElement.nodeName == "DIV") {
+                    rssFeedTable.deselectAll();
+                    rssFeedTable.deselectRow();
+                    rssFeedContextMenu.updateMenuItems();
+                }
+            });
+
+            new ClipboardJS('#CopyFeedURL', {
+                text: () => {
+                    let joined = "";
+                    rssFeedTable.selectedRows
+                        .filter(row => rssFeedTable.rows[row].full_data.dataUid != "")
+                        .each(row => joined += rssFeedTable.rows[row].full_data.dataUrl + '\n');
+                    return joined.slice(0, -1);
+                }
+            });
+            rssFeedTable.setup('rssFeedTableDiv', 'rssFeedFixedHeaderDiv', rssFeedContextMenu);
+
+
+            rssArticleContextMenu = new window.qBittorrent.ContextMenu.RssArticleContextMenu({
+                targets: '.rssArticleElement',
+                menu: 'rssArticleMenu',
+                actions: {
+                    Download: function(el) {
+                        let dlString = "";
+                        rssArticleTable.selectedRows.each(row => {
+                            dlString += rssArticleTable.rows[row].full_data.torrentURL + "\n";
+                        });
+                        showDownloadPage([dlString]);
+                    },
+                    OpenNews: function(el) {
+                        rssArticleTable.selectedRows.each(row => {
+                            window.open(rssArticleTable.rows[row].full_data.link);
+                        });
+                    }
+                },
+                offsets: {
+                    x: -16,
+                    y: -57
+                }
+            });
+            rssArticleTable.setup('rssArticleTableDiv', 'rssArticleFixedHeaderDiv', rssArticleContextMenu);
+            updateRssFeedList();
+            feedRefreshTimer = setInterval(updateRssFeedList, 1000);
```

Comment:
Please move the interval duration to a top-level variable. I also think we should bump it `1500` for consistency with our main data sync.